Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7235,7 +7241,7 @@ void WebPageProxy::didCommitLoadForFrame(IPC::Connection& connection, FrameIdent
         m_pluginMinZoomFactor = { };
         m_pluginMaxZoomFactor = { };
 #if ENABLE(POINTER_LOCK)
-        requestPointerUnlock();
+        requestPointerUnlock([](bool) { });
```

Comment:
Is there a reason we don't do here what we do in the completion handler in WebPageProxy::dispatchActivityStateChange?  Same in WebPageProxy::resetState.  If we need to do the same thing in all 3 places, the code should be shared.